docs/zh/latest/admin-api.md

```diff
@@ -598,7 +598,7 @@ APISIX 的 Upstream 除了基本的负载均衡算法选择外，还支持对上
 
 **upstream 对象 json 配置内容：**
 
-```shell
+```json
```

Comment:
we should use `shell` here.

<img width="703" alt="image" src="https://user-images.githubusercontent.com/6814606/156345849-f48ccac4-23d8-4668-aa03-de0944827bbf.png">